src/lib/raft/raft.c

```diff
@@ -158,6 +159,32 @@ raft_add_vote(struct raft *raft, int src, int dst)
 	return true;
 }
 
+/**
+ * Vote can be revoked if it was volatile (not yet in WAL) and thus wasn't shown
+ * to any other instance yet.
+ */
+static void
+raft_del_vote(struct raft *raft, int src, int dst)
```

Comment:
It seems we are only deleting votes from `raft->self`. Let's remove `src` from parameters then?

Reply:
Done.